src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin: 0 4px 0 0;
+    width: 16px;
+    height: 16px;
+    background-image: url('../images/star.svg');
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &--4 :nth-child(-n + 4) {
```

Comment:
Uuuuh, almost perfect, Mario! use this styling here with a `@for` so you can apply it to any modifier from 0 stars to 5 stars.
But I like your approach very much!